examples/cpma-config.example.yaml

```diff
@@ -1,7 +1,10 @@
 ---
-Source: "master-0.gildub1.lab.pnq2.cee.redhat.com" # can also point to directory
+Source: "master-0.example.com" # can also point to directory
 SSHCreds:
   Login: "root"
-  PrivateKey: "/home/gildub/.ssh/quicklab"
+  PrivateKey: "/home/example/.ssh/key"
   Port: 22
 OutputDir: "./data"
+# Masterf and Nodef are optional fields, use only if cluster was configured with different config locations
+Masterf: "/etc/origin/master/master-config.yaml"
+Nodef: "/etc/origin/node/node-config.yaml"
```

Comment:
Patch looks good. One things before we merge:
- use more user friendly key name. E.g. "MasterConfigFile" is so much better than Masterf.

Reply:
Done